native/pom.xml

```diff
@@ -16,6 +16,31 @@
         <module>mkl</module>
         <module>jni</module>
     </modules>
-
-
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <compilerStartOptionDirectory>-I ${JAVA_HOME}/include/linux/</compilerStartOptionDirectory>
+                <linkerStartOptionDirectory>-I ${JAVA_HOME}/include/linux/</linkerStartOptionDirectory>
+                <packagingDirectory>so</packagingDirectory>
+                <includeDirectory>libiomp5.so</includeDirectory>
+                <typeDirectory>so</typeDirectory>
+                <destFileNameDirectory>libjmkl.so</destFileNameDirectory>
```

Comment:
destFileNameDirectory -> destFileName?

BTW, why you always have a Directory suffix...